application/config/features.php

```diff
@@ -14,11 +14,12 @@
 
 	// Post views
 	'views' => [
-		'map' => TRUE,
+		'map' => FALSE,
 		'list' => TRUE,
 		'chart' => TRUE,
 		'timeline' => TRUE,
 		'activity' => TRUE,
+        'plan' => TRUE,
```

Comment:
If this is cloud only - it should be disabled by default. The default state here should represent the OSS settings.